app/pubspec.yaml

```diff
@@ -120,6 +120,7 @@ dependencies:
   flutter_local_notifications: ^16.0.0
   email_validator: ^2.1.17
   app_settings: ^5.1.1
+  readmore: ^2.2.0
```

Comment:
We can remove this and instead use ```expandable_text``` already existing ;)

Reply:
Yes sure :)